src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -103,14 +122,18 @@ def __call__(
         latents: Optional[torch.FloatTensor] = None,
         output_type: Optional[str] = "pil",
         return_dict: bool = True,
+        output_latents: bool = False,
+        run_safety_checker: bool = True,
+        enable_grad: bool = False,
         **kwargs,
     ):
         r"""
         Function invoked when calling the pipeline for generation.
 
         Args:
-            prompt (`str` or `List[str]`):
-                The prompt or prompts to guide the image generation.
+            prompt (`str`, `List[str]` or `torch.Tensor`):
```

Comment:
Ufff, I don't think the input `prompt` should ever be a tensor that's confusing and opens the box for hacky code - can't we just work with the `latents` inputs?

Reply:
What if we use `inputs_embeds` as some methods in `transformers` have?

Reply:
This is what I meant: https://github.com/huggingface/transformers/blob/16913b3c9215f592f1240511f8da271dc07c3552/src/transformers/models/t5/modeling_t5.py#L905

Reply:
Agree with Patrick.

Reply:
The problem is that we can pass an image to StableDiffusionImg2ImgPipeline.__call__ as a tensor but we can't pass a text... 
Would and extra `inputs_embeds` or `prompt_embeds` argument work as an alternative?